pkg/httputils/http_util.go

```diff
@@ -310,7 +310,7 @@ func GetRangeSE(rangeHTTPHeader string, length int64) ([]*RangeStruct, error) {
 	if strings.ContainsAny(rangeHTTPHeader, "=") {
 		rangeSlice := strings.Split(rangeHTTPHeader, "=")
 		if len(rangeSlice) != 2 {
-			return nil, errors.Wrapf(errortypes.ErrInvalidValue, "invaild range: %s, should be like bytes=0-1023", rangeStr)
```

Comment:
I found that there are two places in this file, which has the same incorrect spelling.
Could you help to fix that in this single pull request? @ethan-daocloud

Reply:
sure, I will find and fix that.

Reply:
@allencloud 
And I check this file thoroughly, find one more , please confirm :)